pkg/drivers/http/element.go

```diff
@@ -248,6 +248,16 @@ func (el *HTMLElement) CountBySelector(selector values.String) values.Int {
 	return values.NewInt(selection.Size())
 }
 
+func (el *HTMLElement) ExistsBySelector(selector values.String) values.Boolean {
+	selection := el.selection.Closest(selector.String())
+
+	if selection == nil {
+		return values.False
+	}
+
+	return values.True
```

Comment:
The same with [this](https://github.com/MontFerret/ferret/pull/210/files#r242466319).